railties/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+*   Add `--skip-bcrypt` option to app generator; enable the corresponding
```

Comment:
The CHANGELOG entry should only describe changes to the end user. Describing the CLI options and maybe how --no-skip overrides --minimal are probably all that is necessary.